marklogic-data-hub/src/main/resources/ml-modules/root/data-hub/5/data-services/ingestion/bulkInputCallerImpl.api

```diff
@@ -0,0 +1,28 @@
+{
+  "endpoint": "/bulkInputCallerImpl.sjs",
```

Comment:
For the name - let's call this "bulkIngester.api" for now. 

Also, try adding a JUnit test for this. MappingTest:runMappingStepViaDataServicesEndpoint is our one and only test that uses the Bulk API. I'm due for writing a guide on testing, but briefly, you can do the following to get started:

- Add a new test under src/test/java/com/marklogic/hub/dataservices/ingestion - BulkIngestTest
- Extend AbstractHubCoreTest, which has a getHubClient() method that you can use to get a DatabaseClient for each database
- Insert 2 docs via the BulkInputCaller and then verify the URI/permissions of each